benchmarks/dynamo/torchbench.yaml

```diff
@@ -252,6 +252,7 @@ accuracy:
     eager_not_deterministic:
       # Models that deterministic algorithms can not be turned on for eager mode.
       - Background_Matting
+      #- pytorch_unet
```

Comment:
@lezcano Check the new test results as I have just commented out this

Reply:
@lezcano It is done so you can check yourself. If we don't enable `pytorch_unet` under this section we have `eager_two_runs_differ`. 
If we add it we need to switch the expected to `pass_due_to_skip`. This is why we have changed all these csv to `pass_due_to_skip`.

Reply:
Is it correct that this is going to impact both eager and non-eager expected results also if it is called `eager_not_deterministic`?

Reply:
This feels a bit annoying, but it also makes sense. Don't we have a list where we can bump the tolerance for a given model? WDYT @ezyang